### PR TITLE
Resolves DCP2-480: ArcLight -Add and reorder EAD sections in side nav…

### PR DESCRIPTION
**Pull request by gkostin1966**
…igation

### CODE REVIEW

File: app/controllers/concerns/umich/catalog.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module Umich
+  module Catalog
+    extend ActiveSupport::Concern
+
+    included do
+      Blacklight::Configuration.define_field_access :biography_history_field, Blacklight::Configuration::ShowField
```
Comment (estelendur) [inline app/controllers/concerns/umich/catalog.rb:6]: I believe you can also do this by adding the new field symbols to `config/initializers/arclight.rb`

**Comment by cmkpowell** [comment]
Do we really need this? Shouldn't the link to Related cover it?  And come to think of it, do we need the heading ADDITIONAL DESCRIPTIVE DATA: at all?

**Review by estelendur — CHANGES_REQUESTED** [review]
I believe we also wanted to place Related/Additional and Subjects below Contents on the page, because they are or can be so very long

**Comment by cmkpowell** [comment]
I am misunderstanding the purpose, I think, and there isn't enough context in the screenshot to judge whether anything has moved.  If we are changing Related to Additional Information and getting rid of the ADDITIONAL DESCRIPTIVE DATA: subhead, I am all for it.

**Comment by gkostin1966** [comment]
> 

I interrupted the ticket differently given ...  
```
This would be our proposed order: 

Summary

Please move Other Finding Aids up here. When this information is included, we don’t want users to miss it. 

Biography/History

Arrangement

Scope and Content

Additional Information [We suggest renaming the Related section]

This could include the other fields from the current Background section, such as Acquisition, Custodial history, Processing information, and Rules or Conventions

Please make sure all <add> elements are displaying. Currently lists, chronologies, and bibliographies still don’t appear. 

Subjects

Contents

Using This Material
```
Which has Contents at the end above Using This Material.

File: app/controllers/concerns/umich/catalog.rb (hunk 1 above)
Comment (gkostin1966) [reply app/controllers/concerns/umich/catalog.rb:6]: Yes, you are correct, I'll delete it from the PR.  Thanks.

**Comment by cmkpowell** [comment]
Okay, I think we need to get back with the original reporter to see if this meets their needs.

**Comment by cmkpowell** [comment]
This on hold until we can share with all stakeholders.

**Comment by gkostin1966** [comment]
Redo

**Pull request closed without merging**